gradle-baseline-java-config/resources/checkstyle/checkstyle.xml

```diff
@@ -127,6 +127,10 @@
             <property name="illegalPkgs" value="junit.framework"/>
             <message key="import.illegal" value="Use JUnit 4-style (org.junit.*) test classes and assertions instead of JUnit 3 (junit.framework.*)."/>
         </module>
+        <module name="IllegalImport"> <!-- Some OpenJDK builds do not include javafx -->
+            <property name="illegalPkgs" value="javafx"/>
+            <message key="import.illegal" value="Must not import javafx classes."/>
```

Comment:
Could you update the description to provide more information describing why it's undesirable to use javafx?

Reply:
done